src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, each time you call the `getRandomColor()` method, a new `Random` object is created. Instead, you should create a single `Random` object and use it throughout the class. Add a private field to the class: `private Random random = new Random();` Then, use this field in your method: `random.nextInt(COLORS_COUNT);`